### PR TITLE
Replace shell calls by rmdir(), support Windows

### PR DESCRIPTION
**Pull request by oleg-nenashev**
@reviewbybees

### CODE REVIEW

**Comment by recena** [comment]
@oleg-nenashev LGTM but there are conflicts.

**Comment by oleg-nenashev** [comment]
@recena Yeah, I pushed merge to the wrong repo

File: TODO.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Before 1.0
 ---
 
 Code:
-* [ ] Replace rm calls by rmDir in workflow 1.11
+* [x] Replace rm calls by rmDir in workflow 1.11
```
Comment (recena) [inline TODO.md]: You means `deleteDir`, right?

Comment (oleg-nenashev) [inline TODO.md]: Right. This file will be deleted during the release in any case, but I'll fix it

**Comment by recena** [comment]
:bee:

**Comment by amuniz** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Pull request merged**